### DISCUSSION TITLE
TL-WN822Nv4 Working confirmed

### DISCUSSION THREAD

**Discussion by sieutungbi** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#101] [Show and tell]
Hello guys, I just want to say thank you all for the hard work, I just install hackintosh 11.5.2 and this save me a lot of time and effort, thank you again.

**Comment by chris1111** [comment]
Great ! Thanks to report.

**Reply by thetrungkien** [reply]
How can you do that (sieutungbi), I just installed Big sur 11.5.2 too with TL-WN823n adapter wifi but even effort a lot of time it's not working (it worked well in Catalina 10.15.7)
